modules/apps/info/info-test/src/testIntegration/java/com/liferay/info/request/struts/test/EditInfoItemStrutsActionTest.java

```diff
@@ -657,6 +657,9 @@ private UploadPortletRequest _getUploadPortletRequest(
 		MockMultipartHttpServletRequest mockMultipartHttpServletRequest =
 			new MockMultipartHttpServletRequest();
 
+		mockMultipartHttpServletRequest.setContentType(
```

Comment:
I keep using mockMultipartHttpServletRequest.setContentType() here becuase

only MockMultipartHttpServletRequest has the method of setContentType(). If I want to add this to my UploadTestUtil api, we would need to cast our uploadServletRequest to MockMultipartHttpServletRequest to use setContentType(). But even though I cast, I still cannot make it work.
I didn't use _getMultipartHttpServletRequest() for setContentType() is because in ExecuteInfoItemActionStrutsActionTest.java, we do not have a method like this. Directly using setContentType("multipart/form-data;boundary=" + System.currentTimeMillis()) is the most clear and easy fix for both classes IMO.